doc/interfaces/serial.rst

```diff
@@ -2,9 +2,18 @@
 
 CAN over Serial
 ===============
-
 A text based interface. For example use over bluetooth with
-``/dev/rfcomm0``
+``/dev/rfcomm0`` or with Arduino ``/dev/ttyACM0``. The interface is a simple
```

Comment:
Let's include a couple of really common serial channels: ``COM1``, ``/dev/ttyUSB0``, ``/dev/ttyS1``.  I don't have a problem linking to your Arduino example, but perhaps a bit later in the docs - after establishing the basics. We want readers to feel confident that this interface will allow them to communicate with anything that can speak serial.

If you feel inspired perhaps you could write a standalone demo in the docs using `socat` and a pseudo term? See e.g. https://stackoverflow.com/questions/52187/virtual-serial-port-for-linux